### ISSUE TITLE
Pitch detection with samplerate > 48khz

### ISSUE THREAD

**Issue by aubiouser12** [aubio/aubio#159]
Hi,
when i'm using aubio pitch detection example with samplerates up to 48khz everything works fine.
Samplerates higher than 48khz, e.g. 96khz or 192khz, doesnt give any pitch results?
Is there a limitation or something special to do?

Thanks in advance

**Comment by piem** [comment]
hi @aubiouser12 

No, there should be no limitations, provided you have built against a library that can do up-sampling. 

Did you build aubio against libsndfile, ffmpeg, or AudioToolbox? Which example are you using? Can you reproduce with `aubiopitch` (c program) or `aubio pitch` (python script)?

cheers, piem

**Comment by aubiouser12** [comment]
hi @piem 
Thank you for your fast reply. I noticed this behaviour on a raspberry pi. I installed aubio with apt (aubio-tools) and also tried it with compiling the latet version by myself without configuring anything. With both versions i had no luck with samplerates higher than 48khz, although i tested it with wav files with the appropriate samplerates.
Yesterday i tried the same thing on a debian linux x86 and everything worked fine, also samplerates higher than 48khz?!

I think i have to look closer whats happening on the raspberry..
